configurations/default/env.yml.tmp

```diff
@@ -75,4 +75,13 @@ SPARKPOST_KEY: your-api-key
 VALIDATE_ENVIRONMENT_CONFIG: true
 
 # The expected frequency to receive live journey location data.
-TRIP_TRACKING_UPDATE_FREQUENCY_SECONDS: 5
\ No newline at end of file
+TRIP_TRACKING_UPDATE_FREQUENCY_SECONDS: 5
+# The minimum segment size in seconds for interpolated points.
+TRIP_TRACKING_MINIMUM_SEGMENT_TIME: 5
+# Trip tracking mode boundaries in meters
+TRIP_TRACKING_WALK_BOUNDARY: 5
+TRIP_TRACKING_BICYCLE_BOUNDARY: 10
+TRIP_TRACKING_BUS_BOUNDARY: 20
+TRIP_TRACKING_SUBWAY_BOUNDARY: 100
+TRIP_TRACKING_TRAM_BOUNDARY: 100
+TRIP_TRACKING_RAIL_BOUNDARY: 200
```

Comment:
Maybe we need an on-schedule threshold too.

Reply:
I don't follow, can you expand on this please?

Reply:
I was thinking we might a time threshold, but I guess because we are tracking the distance to an expected location, the time threshold is probably not needed.